### ISSUE TITLE
EQM - Reports - Getting a 500 error when viewing the 'Difficult questions' for a not started quiz

### ISSUE THREAD

**Issue by pcenov** [learningequality/kolibri#12431]
## Observed behavior

When I go to Coach > Reports > Quizzes and I attempt to preview the 'Difficult questions' for a just created not started quiz I am getting a server error:

```
{
  "data": "\n<!doctype html>\n<html lang=\"en\">\n<head>\n  <title>Server Error (500)</title>\n</head>\n<body>\n  <h1>Server Error (500)</h1><p></p>\n</body>\n</html>\n",
  "status": 500,
  "statusText": "Internal Server Error",
  "headers": {
    "content-length": "145",
    "content-type": "text/html",
    "date": "Tue, 09 Jul 2024 13:39:38 GMT",
    "referrer-policy": "same-origin",
    "server": "Cheroot/unknown",
    "vary": "Cookie",
    "x-content-type-options": "nosniff",
    "x-frame-options": "DENY"
  },
  "config": {
    "transitional": {
      "silentJSONParsing": true,
      "forcedJSONParsing": true,
      "clarifyTimeoutError": false
    },
    "adapter": [
      "xhr",
      "http",
      "fetch"
    ],
    "transformRequest": [
      null
    ],
    "transformResponse": [
      null
    ],
    "timeout": 0,
    "xsrfCookieName": "kolibri_csrftoken",
    "xsrfHeaderName": "X-CSRFToken",
    "maxContentLength": -1,
    "maxBodyLength": -1,
    "env": {},
    "headers": {
      "Accept": "application/json, text/plain, */*",
      "X-Requested-With": "XMLHttpRequest"
    },
    "paramsSerializer": {},
    "url": "/coach/api/quizdifficulties/26/",
    "params": {
      "classroom_id": "ded7bae236498996c848802a0a39540e"
    },
    "method": "get"
  },
  "request": {}
}
```

Video:

https://github.com/learningequality/kolibri/assets/79847249/80fff39a-5c7b-4369-9851-b34b858e6966

## Expected behavior

The user should see the difficult questions table with text: There are no questions

## Steps to reproduce the issue

1. Install the build from https://github.com/learningequality/kolibri/releases/tag/v0.17.0-beta0
2. Create a new quiz with sections and questions and save it.
3. While at Coach > Reports > Quizzes click on the title of a quiz and then select the 'Difficult questions' tab

## Usage Details
Windows 11, Ubuntu 22  - Chrome

**Comment by pcenov** [comment]
@radinamatic

**Comment by nucleogenesis** [comment]
After chatting with @radinamatic & @tomiwaoLE about what I found while looking into this we figured the most reasonable solution is to just not show the "Difficult questions" tab when the quiz has not yet been started.

Basically, if you have never started a quiz, then it can never have had any answers given so there cannot be difficult questions. Hiding or disabling that tab on reports pages should do it here.
